### ISSUE TITLE
Bugs in Local preferences in cooperation with screen readers

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#1527]
**Reported by krzyszp on 23 May 41858558 18:17 UTC**
In Local preferences (advanced view) labels for text fields are not readable by NVDA (screen reader). (Manager 6.10.13)

Migrated-From: http://boinc.berkeley.edu/trac/ticket/1150
